docs/DeveloperGuide.md

```diff
@@ -182,12 +183,57 @@ This section describes some noteworthy details on how certain features are imple
 
 ##### Implementation
 
+#### 2.3 Show event
+
+#### Implementation
+
+The following is a detailed explanation of the operations which take place when the `showevents` command is called
```

Comment:
Would it be easier on the reader if you used a sequence diagram perhaps?

Reply:
Added!